package-lock.json

```diff
@@ -2954,7 +2954,7 @@
     "minimatch": {
       "version": "3.0.4",
       "resolved": "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz",
-      "integrity": "sha512-yJHVQEhyqPLUTgt9B83PXu6W3rx4MvvHvSUvToogpwoGDOUQ+yDrR0HRot+yOCdCO7u4hX3pWft6kWBBcqh0UA==",
+      "integrity": "sha1-UWbihkV/AzBgZL5Ul+jbsMPTIIM=",
```

Comment:
:question: @davidmsibley what npm version are you running?
the integrity hash is being downgraded from `sha512` to `sha1`.

Reply:
weird. 5.3.0. I'll upgrade.

Reply:
I had that happen to me once, upgrading fixed that.

Reply:
upgraded to 5.5.1 and it still decided to chop it down to sha1

Reply:
ah. I may need to delete node_modules first.